GHA Windows version 20201116.1 breaks build

The GHA Windows build has started failing on `master`:

```
Finished generating code
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Tools\\MSVC\\14.28.29333\\bin\\HostX86\\x86\\link.exe' failed with exit status 1257
```

Last pass: https://github.com/python-pillow/Pillow/actions/runs/370215236

Version: 20201108.1
Included Software: https://github.com/actions/virtual-environments/blob/win19/20201108.1/images/win/Windows2019-Readme.md

First fail: https://github.com/python-pillow/Pillow/actions/runs/376064850

Version: 20201116.1
Included Software: https://github.com/actions/virtual-environments/blob/win19/20201116.1/images/win/Windows2019-Readme.md

Change included software: https://github.com/actions/virtual-environments/commit/9a16e36b2847d6e56ba0ff3c0bd275674f1988cc#diff-816219d3c9f30fcc1c40f88ca826c1662bf99eded781c1b419dd2dfb577ae227

First error:

```
2020-11-21T12:51:28.1169099Z ConvertYCbCr.c
2020-11-21T12:51:28.3157967Z fatal error C1047: The object or library file 'D:\a\Pillow\Pillow\winbuild\build\lib\lcms2_static.lib' was created by a different version of the compiler than other objects like 'build\temp.win-amd64-3.6\Release\src\_imagingcms.obj'; rebuild all objects and libraries with the same compiler
2020-11-21T12:51:28.3164554Z LINK : fatal error LNK1257: code generation failed
```

--- a/.github/workflows/test-windows.yml
+++ b/.github/workflows/test-windows.yml
@@ -91,25 +91,32 @@ jobs:
     - name: Build dependencies / libjpeg-turbo
       if: steps.build-cache.outputs.cache-hit != 'true'
       run: "& winbuild\\build\\build_dep_libjpeg.cmd"
+
     - name: Build dependencies / zlib
       if: steps.build-cache.outputs.cache-hit != 'true'
       run: "& winbuild\\build\\build_dep_zlib.cmd"
+
     - name: Build dependencies / LibTiff
       if: steps.build-cache.outputs.cache-hit != 'true'
       run: "& winbuild\\build\\build_dep_libtiff.cmd"
+
     - name: Build dependencies / WebP
       if: steps.build-cache.outputs.cache-hit != 'true'
       run: "& winbuild\\build\\build_dep_libwebp.cmd"
+
     # for FreeType CBDT font support
     - name: Build dependencies / libpng
       if: steps.build-cache.outputs.cache-hit != 'true'
       run: "& winbuild\\build\\build_dep_libpng.cmd"
+
     - name: Build dependencies / FreeType
       if: steps.build-cache.outputs.cache-hit != 'true'
       run: "& winbuild\\build\\build_dep_freetype.cmd"
+
     - name: Build dependencies / LCMS2
       if: steps.build-cache.outputs.cache-hit != 'true'
       run: "& winbuild\\build\\build_dep_lcms2.cmd"
+
     - name: Build dependencies / OpenJPEG
       if: steps.build-cache.outputs.cache-hit != 'true'
       run: "& winbuild\\build\\build_dep_openjpeg.cmd"
@@ -123,9 +130,11 @@ jobs:
     - name: Build dependencies / HarfBuzz
       if: steps.build-cache.outputs.cache-hit != 'true'
       run: "& winbuild\\build\\build_dep_harfbuzz.cmd"
+
     - name: Build dependencies / FriBidi
       if: steps.build-cache.outputs.cache-hit != 'true'
       run: "& winbuild\\build\\build_dep_fribidi.cmd"
+
     - name: Build dependencies / Raqm
       if: steps.build-cache.outputs.cache-hit != 'true'
       run: "& winbuild\\build\\build_dep_libraqm.cmd"

